src/sema/ast.rs

```diff
@@ -838,8 +838,10 @@ pub enum Expression {
     },
     StructLiteral {
         loc: pt::Loc,
+        id: pt::IdentifierPath,
         ty: Type,
-        values: Vec<Expression>,
+        // pt::Identifier represents the field name
```

Comment:
I think this should be a doc-comment, it's quite relevant information that should show up in help texts from the analyzer when using this Enum